alchemy/src/supabase/function.md

```diff
@@ -0,0 +1,220 @@
+# Supabase Function Resource
+
+The `Function` resource manages Supabase Edge Functions, which provide serverless compute capabilities at the edge.
+
+## Usage
+
+```typescript
+import { Function } from "alchemy/supabase";
+
+// Create a new Edge Function
+const func = Function("hello-world", {
+  projectRef: "proj-123",
```

Comment:
Would be better if the examples referenced an actual Project resource 

Instead of projectRef: string, it should be project: string | Project so both are supported